.github/workflows/release-package.yml

```diff
@@ -0,0 +1,92 @@
+on:
+  push:
+    #
+    # For push events to refs/tags matching the pattern
+    # 'release/*' - i.e. 'release/6.0.2.1', 'release/bogus'
+    #
+    tags:
+    - 'release/*'
+
+jobs:
+  #
+  # Build and release debian package on Github.
+  # Note that we require Python 3 to be installed
+  # for # the Python bindings to be compiled in
+  # the package.
+  #
+  deb-build-test-release:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      - name: Install debian utilities
+        run: sudo apt-get install -y debhelper devscripts equivs
+      - name: Build dependencies non-interactively
+        run: sudo mk-build-deps --install --tool='apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes' debian/control
+      - name: Build and generate package
+        run: dpkg-buildpackage -b -us -uc
```

Comment:
I think we may want to eventually move this into `delphix/actions` so it can be easily used by other projects, but until we have a second project needing this, I'm good as-is.. with that said, we're already using it twice in this repository..

Reply:
I 100% agree but as you said let's wait how this goes for this repo and then when we have a second project (probably `drgn` or `sdb`) we can re-iterate then.